### PR TITLE
Change lexical scope led_strip

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: src/js/tabs/led_strip.js [hunk 1]
```diff
@@ -344,33 +344,19 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
...
                         // Get function & overlays or current cell
                         TABS.led_strip.directions.forEach(function(letter) {
-                            var className = '.dir-' + letter;
+                            const className = '.dir-' + letter;
                             if ($(that).is(className)) {
                                 directionsInSelection.push(className);
                             }
                         });
-
-                        TABS.led_strip.baseFuncs.forEach(function(letter) {
```
Comment (mikeller) [inline src/js/tabs/led_strip.js:353]: Where did these go?

File: src/js/tabs/led_strip.js [hunk 2]
```diff
@@ -394,19 +380,19 @@ TABS.led_strip.initialize = function (callback, scrollPosition) {
 
                     // set checkbox values
                     TABS.led_strip.overlays.forEach(function(letter) {
-                        var feature_o = $('.checkbox').find('input.function-' + letter);
+                        const featureO = $('.checkbox').find(`input.function-${letter}`);
 
-                        var newVal = ($(uiSelectedLast).is('.function-' + letter));
+                        const newVal = ($(uiSelectedLast).is(functionTag + letter));
 
-                        if (feature_o.is(':checked') != newVal) {
```
Comment (mikeller) [inline src/js/tabs/led_strip.js:401]: What does the `O` in the name stand for?

File: src/js/tabs/led_strip.js (hunk 1 above)
Comment (haslinghuis) [reply src/js/tabs/led_strip.js:353]: Not sure why I removed them. There was no change in functionality. To be sure added them back and there seems no difference. This tab is too complicated and was the hardest to fix.

File: src/js/tabs/led_strip.js (hunk 2 above)
Comment (haslinghuis) [reply src/js/tabs/led_strip.js:401]: @McGiverGim do you know?

Comment (McGiverGim) [reply src/js/tabs/led_strip.js:401]: I don't know... maybe overlay?

**Review by mikeller — APPROVED** [review]

**Pull request merged**